packages/sanes-browser-extension/src/extension/background/model/persona/config/codec.ts

```diff
@@ -54,22 +54,19 @@ export function pathBuilderForCodec(codecType: CodecType): (derivation: number)
   return pathBuilder;
 }
 
-export function chainConnector(
+export async function chainConnector(
```

Comment:
Why async?

Reply:
You are right, I had used another way but forgot to remove this `async`